{"uuid": "33a34d48-827a-42d9-b0fe-8304ebd97a41", "children": ["8763ab48-b06c-4f73-a584-1818f9e80ecb", "7ee2241a-aaa3-4bd2-ab2b-54b116498a38", "7f27919d-de2e-414a-830a-205f3ef484e5", "03b8332e-9c0d-4da8-9f8e-44efe27d19e7", "75f67799-b0a9-4d48-ac30-d716fdac2cb8", "0093d088-1dc8-4fc3-bf35-503466d1c72f", "28ed5815-3b8b-4e7d-8986-303121623227", "764cf3b9-765d-4929-a33b-169585bdfbeb", "b124fe14-e4c7-4a09-8ea3-3bf58f880861"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for safari; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\n", "trace": "  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\tests\\conftest.py\", line 30, in browser\n    driver = webdriver.Safari()\n             ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\selenium\\webdriver\\safari\\webdriver.py\", line 48, in __init__\n    self.service.path = self.service.env_path() or DriverFinder(self.service, options).get_driver_path()\n                                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py\", line 50, in get_driver_path\n    return self._binary_paths()[\"driver_path\"]\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Katherine Gomez\\Documents\\Documentos\\QA\\Selenium y Python\\TestEnv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py\", line 78, in _binary_paths\n    raise NoSuchDriverException(msg) from err\n"}, "start": 1733970638836, "stop": 1733970642416}], "start": 1733970638836, "stop": 1733970643084}